setup.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+from setuptools import setup
+
+requires = []
+
+entry_points = {
+    'console_scripts': []
+}
+
+packages = []
+
+setup(
+    name="pelican-tools",
+    version="1.0.0",
+    url='http://getpelican.com/',
+    author='Alexis Metaireau',
+    author_email='authors@getpelican.com',
+    description="Tools developed for the Pelican static site generator.",
```

Comment:
Good point; I've addressed this and rebased. Seems like this can be changed in the main Pelican repository as well.